app/src/main/java/com/ataulm/whatsnext/WhatsNextRepository.kt

```diff
@@ -112,3 +118,24 @@ private class PopularFilmsThisWeekPagingSource(
         )
     }
 }
+
+private class WatchListPagingSource(
+        private val memberId: String,
+        private val letterboxdApi: LetterboxdApi,
+        private val filmSummaryConverter: FilmSummaryConverter
+) : PagingSource<String, FilmSummary>() {
+
+    override suspend fun load(params: LoadParams<String>): LoadResult<String, FilmSummary> {
+        val memberId = letterboxdApi.me().member.letterboxdId // TODO: don't leave this here
```

Comment:
We should be able to load anyone's watchlist (if they haven't set it to private) so hardcoding here isn't great. It's also wasteful since our ID isn't going to change.